docs/docs-rules/design/05-test-strategy.md

```
@@ -205,6 +205,23 @@ tests/unit/[path]/[methodName]/
 └── mocks/
     └── createMock[ClassName].ts    # モックファクトリ（既存モックがない場合のみ新規作成）
 \`\`\`
+
+### テストヘルパー
+
+テストヘルパークラスを使用する場合、**実装コードは記載せず、ファイルパスの参照のみ**とする。
+
+\`\`\`markdown
+### テストヘルパー
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)